frame/session/src/lib.rs

```diff
@@ -170,6 +170,14 @@ impl<
 			offset
 		})
 	}
+
+	fn weight(_now: BlockNumber) -> Weight {
+		// `estimate_next_session_rotation` has no storage reads and trivial computational overhead.
+		// There should be no risk to the chain having this weight value be zero for now.
+		// However, this value of zero was not properly calculated, and so it would be reasonable
```

Comment:
If you think this is important, issue + a `TODO` would be better. 

You could also write it down as `// Weight note: ...` and if we always put such comments with this prefix, they will be easy to find later on.

Reply:
i dont think it is important

Reply:
added a tag